base/setup/usetup/mui.h

```diff
@@ -71,6 +80,258 @@ LPSTR
 MUIGetString(
     ULONG Number);
 
+VOID
+MUIGetEntryData(
+    IN ULONG page,
+    IN UINT TextID,
+    IN MUI_ENTRY_DATA EntryData,
+    OUT PVOID OutputData);
+
+/* MUI Text IDs */
+
+/* General ReactOS Banner Version Text ID */
+#define TEXT_REACTOS_BANNER 1
```

Comment:
OK I think actually we can do something similar to what one usually does for Win32 applications:
For static text, usually we always use the same control ID (== IDC_STATIC == -1 as convention in Win32).
Only static text controls that need to be changed programatically / hidden / etc... (exactly as for the "format" text string we wanted to correctly hide during the formatting step here), should have a specific control ID.
So for our situation at hand we could just use the same "control" ID for all the MUI_ENTRies, except for those we want to control programmatically. This should dramatically reduce the number of IDs you have added there (and potentially make simpler to add the values in the translation files).